### ISSUE TITLE
Posy Strokeless cursor

### ISSUE THREAD

**Issue by Lianaja32** [simtrami/posy-improved-cursor-linux#10]
I think for the link cursor it's suppose to be an arrow with the blue at the top, not the hand. The image shown is the correct link cursor.
![posy-strokeless-link-64](https://github.com/simtrami/posy-improved-cursor-linux/assets/79010647/38f28dbc-e579-483a-ba06-aa55d340b701)

**Comment by simtrami** [comment]
Hi @Lianaja32 ,
I agree with you, the blue cursor should be the one displayed when hovering a link. I just checked, and the [link.bin](https://github.com/simtrami/posy-improved-cursor-linux/blob/8c39b915bbba4717a450ef750fa8b0c0b60cb160/Posy_Cursor_Strokeless/cursors/link) file is actually the one in your image.

Could you provide more information on the issue you're having with this cursor : your OS version, the way you installed the set or steps to reproduce, please ?

**Comment by Lianaja32** [comment]
I am running Ubuntu Budgie 22.04.3 (kernel 6.2.0-37-generic) and I installed the set by clicking code, then download it. And after extracting the set, I copy the Strokeless set to /usr/share/icons. it's when I hover a link I discovered it was the hand cursor, not the blue arrow.

**Comment by simtrami** [comment]
That's weird... While I create a Budgie VM to reproduce your bug, could you send me the SHA256 checksum of the link and the hand files located in `Posy_Cursor_Strokeless/cursors/link`, please?
They should be 

```shell
$ shasum -a 256 link
e4ec343cde58ebe290129b418819f9480f2e5a4111b0eb197b399374086ec420  link
$ shasum -a 256 hand
2d0deba9209af248a6fb70fb24e6e3ad76b49524ced0acca86e84d40b167c8fd  hand
```

**Comment by Lianaja32** [comment]
Link is: e4ec343cde58ebe290129b418819f9480f2e5a4111b0eb197b399374086ec420  link
Hand is: 2d0deba9209af248a6fb70fb24e6e3ad76b49524ced0acca86e84d40b167c8fd  hand

**Comment by simtrami** [comment]
Ok, so I managed to reproduce what you're experiencing, I think.

![Screenshot_budgie](https://github.com/simtrami/posy-improved-cursor-linux/assets/32884998/6c109dd0-6b07-4ba7-bb51-16600e1a7b2c)

This hand cursor is the one from the previously loaded cursor set. This bug is common when you just changed to a new cursor set but then disappears after restarting the program (Firefox, here), the user session or the PC.

However, for me and probably for you too, it persists after a reboot : that usually means the GUI (Budgie) is trying to load a cursor that my set does not have and loads the one in cache (or something like this).

I am going to check which files the default Budgie cursor set have that I don't, add them, check if that fixes things and, eventually, push an update.

**Comment by Lianaja32** [comment]
No, when I hover a link, it load the default Posy's hand cursor.

**Comment by Lianaja32** [comment]
Here is the content of `/usr/share/icons`
![Capture d’écran du 2023-12-12 20-38-33](https://github.com/simtrami/posy-improved-cursor-linux/assets/79010647/74915a0c-5328-48aa-a31a-cd4ea68c08e6)
As for cursors, I can select Bibata-Modern-rainbow, DMZ-Black, DMZ-White and the Posy Cursors pack (I install the entire set).

**Comment by simtrami** [comment]
The set named "Posy's Cursor" is actually the _classic_ one. Do you have "Posy's Cursor Strokeless" in the list in Budgie Desktop Settings?

**Comment by Lianaja32** [comment]
Yes, I'm using it as we speak.

**Comment by simtrami** [comment]
Do you only have the strokeless one in the list or do all the other ones appear also? I am confused about this, can you take a screenshot of the Budgie Desktop Settings window?

**Comment by Lianaja32** [comment]
I have all of the other one but I can't do a screenshot while the selection list is open...

**Comment by Lianaja32** [comment]
file:///home/oratoria/Images/Capture%20d%E2%80%99%C3%A9cran%20du%202023-12-12%2021-20-21.png

**Comment by Lianaja32** [comment]
![Capture d’écran du 2023-12-12 21-20-21](https://github.com/simtrami/posy-improved-cursor-linux/assets/79010647/e9cb16d8-7a22-4de6-bf6b-a311cc0c5d76)

**Comment by Lianaja32** [comment]
Maybe inverting the roles will be fine (the blue arrow for the link cursor and the hand when you drop a file).

**Comment by Lianaja32** [comment]
I moved to Ubuntu Unity 23.10 and the issue is persisting...
Neofetch: 
![Capture d’écran du 2023-12-18 00-10-34](https://github.com/simtrami/posy-improved-cursor-linux/assets/79010647/4954c1da-2bb6-4be6-9a7f-b6b032241dae)
Settings (Unity Tweak Tool): 
![Capture d’écran du 2023-12-18 00-06-25](https://github.com/simtrami/posy-improved-cursor-linux/assets/79010647/cc57c274-5299-4717-a8ec-6f85b7884f8d)

**Comment by simtrami** [comment]
Hi @Lianaja32,
I did not manage to reproduce the bug you have. However, I just pushed an update (bd2bac08bf01e25846a6643dd30e2acffa9517d4) which adds cursor files the DMZ set had that mine did not. I only added them for the Strokeless set yet, so that you can test if it changes anything for you.

I am sorry for the time it takes to fix this, I don't have a lot of time for my open-source projects and mouse cursor set packaging was not something I had plan on doing in the long term when I shared this repo.

**Comment by Lianaja32** [comment]
It's OK mate, take your time.

**Comment by Lianaja32** [comment]
Another recording, this time with Stacer. Keep an eye on the cursor.

https://github.com/simtrami/posy-improved-cursor-linux/assets/79010647/62c55fcf-dac4-41b9-b737-502f9cdba288
